WWDC/SessionCollectionViewItem.swift

```diff
@@ -44,14 +44,10 @@ final class SessionCollectionViewItem: NSCollectionViewItem {
 
     override func viewDidLoad() {
         super.viewDidLoad()
-
```

Comment:
I prefer a vertical whitespace separating the call to super from the rest of the function implementation.